syncd/ComparisonLogic.cpp

```diff
@@ -2836,6 +2838,21 @@ void ComparisonLogic::cretePreMatchForAclEntries(
                 continue;
             }
 
+            if (cPrio->getStrAttrValue() == PFC_ACL_RULE_PRIORITY)
```

Comment:
Do we know that only PFC ACLs have the priority of `PFC_ACL_RULE_PRIORITY`? If not, then there is a chance that this block will get executed for other ACLs, and in that case the behavior is indeterministic.

Reply:
this is hardcoded in OA https://github.com/sonic-net/sonic-swss/blob/f31ccd09b8eafa2d9292e7446fc5fe8d4ec0ad06/orchagent/pfcactionhandler.cpp#L448